src/material.cc

```diff
@@ -53,6 +53,34 @@ void Material::Record(Context* ctx) const {
   comp_->Record(ctx);
 }
 
+std::string Material::GetNormalizedCompAtom() const {
+  std::string comp = "{";
+  cyclus::CompMap c = comp_->atom();
+  cyclus::compmath::Normalize(&c, 1);
+  for (std::map<const int, double>::const_iterator it = c.begin();
+       it != c.end(); ++it) {
+    comp = comp + std::string("{") + std::to_string(it->first) +
+           std::string(",") + std::to_string(it->second) + std::string("},");
+  }
+  comp.pop_back();
+  comp = comp + std::string("}");
+  return comp;
```

Comment:
Most of the code for the atom and mass versions are identical, with just a different `CompMap`, so probably want to combine those into a single method that just gets called with a different `CompMap`.